com.sap.cloud.lm.sl.cf.process/src/test/java/com/sap/cloud/lm/sl/cf/process/jobs/FlowableOperationsCleanerTest.java

```diff
@@ -0,0 +1,68 @@
+package com.sap.cloud.lm.sl.cf.process.jobs;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.flowable.engine.impl.persistence.entity.ExecutionEntityImpl;
+import org.flowable.engine.runtime.ProcessInstance;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+
+import com.sap.cloud.lm.sl.cf.process.flowable.FlowableFacade;
+
+public class FlowableOperationsCleanerTest {
+
+    @Mock
+    private FlowableFacade flowableFacade;
+
+    private FlowableOperationsCleaner flowableOperationsCleaner;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.initMocks(this);
+        flowableOperationsCleaner = new FlowableOperationsCleaner(flowableFacade);
+    }
+
+    static Stream<Arguments> testDeleteInvocation() {
+        // @formatter:off
+             return Stream.of(
+                              Arguments.of(Arrays.asList("process-id-1"), 1),
+                              Arguments.of(Arrays.asList("process-id-1", "process-id-2"), 2),
+                              Arguments.of(Collections.emptyList(), 0)
+             );
+        // @formatter:on
+    }
+
+    @ParameterizedTest
+    @MethodSource
+    void testDeleteInvocation(List<String> processIds, int deleteTimes) {
+        prepareFlowableFacade(processIds);
+        flowableOperationsCleaner.execute(null);
```

Comment:
Pass some real `Date` object here and test whether it's being passed to the Flowable facade.